ui/src/app/workflows/components/workflows-toolbar/workflows-toolbar.tsx

```diff
@@ -10,66 +11,71 @@ require('./workflows-toolbar.scss');
 
 interface WorkflowsToolbarProps {
     selectedWorkflows: Map<string, Workflow>;
-    loadWorkflows: () => void;
-    isDisabled: Actions.OperationDisabled;
+    actionsIsDisabled: Actions.OperationDisabled;
     clearSelection: () => void;
 }
 
-interface WorkflowGroupAction extends WorkflowOperation {
-    groupIsDisabled: boolean;
-    className: string;
-    groupAction: () => Promise<any>;
+interface WorkflowsOperation extends WorkflowOperation {
+    isDisabled: boolean;
+    workflowsAction: () => Promise<any>;
```

Comment:
```suggestion
    action: () => Promise<any>;
```

I think we can just overwrite `action`, since it's not used otherwise. the interface is already plural Workflows, so I don't think we need to repeat it again inside of it

Reply:
Umm Then, Should we rename the name of`action` in below the code?  

- Before

```typescript
        <div className={`workflows-toolbar ${numberSelected === 0 ? 'hidden' : ''}`}>
            <div className='workflows-toolbar__count'>
                {numberSelected === 0 ? 'No' : numberSelected}
                &nbsp;workflow{numberSelected === 1 ? '' : 's'} selected
            </div>
            <div className='workflows-toolbar__actions'>
                {operations.map(action => {
                    return (
                        <button
                            key={action.title}
                            onClick={() => {
                                action.workflowsAction();
                            }}
                            className={`workflows-toolbar__actions--${action.title} workflows-toolbar__actions--action`}
                            disabled={numberSelected === 0 || action.isDisabled}>
                            <i className={action.iconClassName} />
                            &nbsp;{action.title}
                        </button>
                    );
                })}
            </div>
        </div>
```

- After

```typescript
<div className={`workflows-toolbar ${numberSelected === 0 ? 'hidden' : ''}`}>
            <div className='workflows-toolbar__count'>
                {numberSelected === 0 ? 'No' : numberSelected}
                &nbsp;workflow{numberSelected === 1 ? '' : 's'} selected
            </div>
            <div className='workflows-toolbar__actions'>
                {operations.map(operation => {
                    return (
                        <button
                            key={operation.title}
                            onClick={() => {
                                operation.action();
                            }}
                            className={`workflows-toolbar__actions--${operation.title} workflows-toolbar__actions--action`}
                            disabled={numberSelected === 0 || operation.isDisabled}>
                            <i className={operation.iconClassName} />
                            &nbsp;{operation.title}
                        </button>
                    );
                })}
            </div>
        </div>
```

Reply:
Yup, that would be required

Reply:
Oh I missed it, thank you!

Reply:
> Note that with the `onClick` optimization in my [above comment](https://github.com/argoproj/argo-workflows/pull/12046#discussion_r1372339282), those lines become `onClick={operation.action}`

Oh I missed it, thank you!